test/e2e/storage/testsuites/disruptive.go

```diff
@@ -47,11 +46,14 @@ func InitDisruptiveTestSuite() TestSuite {
 		},
 	}
 }
-
 func (s *disruptiveTestSuite) getTestSuiteInfo() TestSuiteInfo {
 	return s.tsInfo
 }
 
+func (s *disruptiveTestSuite) skipUnsupportedSuite(driver TestDriver, pattern testpatterns.TestPattern) {
+	skipVolTypePatterns(pattern, driver)
```

Comment:
At least CSI has different code for in-line volumes and PVs. We can still skip pre-provisioned PVs.

Reply:
I will look into making which VolTypes to skip more flexible